# Define the goal
geometry_msgs/PoseStamped[] grasp_poses
float32 grasp_height 
float32 safety_distance
bool use_map
---
# Define the result
bool success
---
# Define a feedback message
float32 percent_complete